deploy/backplane/rhobs/config.yaml

```diff
@@ -0,0 +1,5 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchLabels:
+      api.openshift.com/rhobs: "true"
```

Comment:
Can someone from the reviewers help me what to put here? The line is just a wild guess for now.